website_anchor_smooth_scroll/static/src/js/website_anchor_smooth_scroll.js

```diff
@@ -0,0 +1,29 @@
+/* © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+odoo.define('website_anchor_smooth_scroll.website_anchor_smooth_scroll', function (require) {
+
+require('web.dom_ready');
+"use strict";
+
+function website_anchor_smooth_scroll(event) {
+    event.preventDefault();
+
+    if (event.target.hash != "") {
+      var target = $(event.target.hash);
+      return $('html, body')
+      .stop()
```

Comment:
not blocking but these lines should be indented under the return statement

Reply:
Ok done, you're right :)